server/services/discord/actions/on_member_joining.js

```diff
@@ -1,11 +1,17 @@
 'use strict'
 
 const client = require('../init').client
+const { AreaGlue } = require('../../glue/glue.js')
 
 /**
  * @brief Triggered when a new user joined the guild.
  * @param {*} user User object
  */
 client.on('guildMemberAdd', user => {
-  console.log('A new user just joined the Discord server: ' + user.tag)
+  try {
+    console.log('A new user just joined the Discord server: ' + user.tag)
```

Comment:
LOG

Reply:
Not mine. It was already there. Should I delete it ?